src/libs/actions/Policy/ReportFields.ts

```diff
@@ -27,6 +27,13 @@ Onyx.connect({
     },
 });
 
+let allReports: OnyxCollection<Report>;
+Onyx.connect({
```

Comment:
@dominictb i don't think we should connect to the same key again. It's already done in ReportUtils

Reply:
thank you, i did not know that